tests/test_merge.py

```diff
@@ -183,21 +179,20 @@ def test_merge_destination_1(tmp_path):
         from rasterio import windows
 
         with rasterio.open(tmp_path.joinpath("test.tif"), "w", **profile) as dst:
-            for _, chunk_window in _chunk_output(
-                dst.width, dst.height, dst.count, 64, mem_limit=1
-            ):
-                chunk_bounds = windows.bounds(chunk_window, dst.transform)
+            for _, chunk in window_split(dst.height, dst.width, max_pixels=256 * 256):
```

Comment:
`window_split()` exists in the test now, but I think it could be welcome in windows.py.